packages/lesswrong/themes/eaTheme.js

```diff
@@ -186,6 +189,16 @@ const theme = createLWTheme({
         fontSize: '50%',
       },
     },
+    PostsTopSequencesNav: {
+      root: {
+        marginBottom: -8,
+      },
+      title: {
+        textTransform: 'lowercase',
```

Comment:
Lowercase sure is a choice here. Surprised by that, but your call.

Reply:
It's then smallcapsed

Reply:
Oh, I see.

Reply:
![image](https://user-images.githubusercontent.com/10352319/73882978-ee703480-4817-11ea-99bc-42d05f71f333.png)

Reply:
Ah, I see. I think I prefer SmallCaps with variations for actual capital letters, but I can see the choice.